tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+	"compilerOptions": {
+		"strictNullChecks": true
+	}
```

Comment:
Are you sure tsconfig.json in packages are taken into account? That's news to me. Any docs about it?

You also need to then add the file to `files` in package.json.

Reply:
Are you talking about a `tsconfig` file being resolved from within node_modules?

Reply:
Yes

Reply:
I don't think so. I believe I found why TSD is enabling `strict` in this [comment](https://github.com/sindresorhus/detect-newline/pull/9#issuecomment-546149192)